lib/pharos/kube/config.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'pharos/yaml_file'
+
+module Pharos
+  module Kube
+    # Loads and manipulates kubeconfig files
+    class Config
```

Comment:
BTW there's also a `K8s::Config` now, but it's not necessarily relevant here... because it's based on `Dry::Struct`, that also means that it's read-only... it's not really designed to write or modify configs.

Reply:
K8s yardocs have `@param config [Pharos::Kube::Config]` in several places